dev/plugins/hu.elte.txtuml.diagnostics/src/hu/elte/txtuml/diagnostics/session/DiagnosticsPlugin.java

```diff
@@ -23,15 +25,17 @@
 
 	private static final int SERVER_SOCKET_BACKLOG = 50;
 	private static final int FAULT_TOLERANCE = 99;
+	private static AtomicInteger DELAY = new AtomicInteger(1000);
```

Comment:
Delay shouldn't look like a constant, the whole point of this feature is to make it configurable.
Suggested declaration: `private AtomicInteger delay = ...`

Reply:
My request above still holds.